lib/rules/at-rule-descriptor-no-unknown/README.md

```diff
@@ -15,6 +15,8 @@ This rule considers descriptors defined in the CSS Specifications, up to and inc
 
 You can filter the [CSSTree Syntax Reference](https://csstree.github.io/docs/syntax/) to find out what descriptors are known for an at-rule.
 
+This rule is only appropriate for CSS. You should not turn it on for CSS-like languages, such as SCSS or Less.
```

Comment:
[suggestion] Do you think marking the paragraph as a warning `[!WARNING]` makes sense?

Reply:
Let's leave it flat and informational because there are other bits in the extended description, like overlaps or what the rule ignores, that could be considered warnings.